scripts/amd/gemm/tune_gemm.py

```diff
@@ -768,8 +887,9 @@ def main():
                 os.remove(generated_script)
                 if not skipWarmup:
                     os.remove(generated_script + ".failed_configs")
-                for f in glob.glob(f"results-{i}.*"):
-                    os.remove(f)
+                for f in glob.glob(f"results-{generated_script}.*"):
+                    if not f.endswith(".csv") and keepRocprof:
```

Comment:
What do you want to keep here? It seems that we will always save *.csv?

Reply:
It seems like I merge this change from somewhere, just reverted it.